client/setup/index.ts

```diff
@@ -0,0 +1,22 @@
+import page from 'page';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { siteSelection, sites } from 'calypso/my-sites/controller';
+import { setupSite } from './setup-site/controller';
+import { STEP_SLUGS } from './setup-site/types';
+
+export default function (): void {
+	// Forgot to specify a site, show a UI to select one
+	page( '/setup', sites, makeLayout, clientRender );
```

Comment:
This isn't a bridge we need to cross right now, but I wanted to elaborate to have it in your mind:

I could also imagine a world where;
- if an existing site is selected, we modify that
- if no site, we assume to create a new site

When re-landing in the site-setup flow before finishing it from last time, we would then ask what customer wants to do:

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/87168/144228983-837ec92f-53d5-40ad-a1ae-b4fe7525affc.png">